app.js

```diff
@@ -1,54 +1,72 @@
-const books = document.querySelector('#book-list');
+const bookSel = document.querySelector('#book-list');
 const button = document.querySelector('#add-book');
-const title = document.querySelector('#title');
-const author = document.querySelector('#author');
-
-// constructor function
-const Book = function (title, author) {
-  this.title = title;
-  this.author = author;
-};
-
-// array of books
+const titleSel = document.querySelector('#title');
+const authorSel = document.querySelector('#author');
 let bookList = [];
 
-const addBookToList = (newBook) => {
+// add books to the page
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.